File: Plot-the-Pattern/B_Chetanya_loves_triangles/Chetanya_loves_triangles.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int rows;
    cin >> rows;

    for (int i = 1; i <= rows; i++)
    {
        for (int j = 2 * rows - i; j > i; j--)
        {
            cout << " ";
        }

        for (int j = i; j >= 1; j--)
        {
            if (j == i || j == 1)
            {
                cout << "* ";
            }
            else
            {
                cout << "  ";
            }
        }

        cout << endl;
    }

    for (int i = rows - 1; i >= 1; i--)
    {
        for (int j = 2 * rows - i; j > i; j--)
        {
            cout << " ";
        }

        for (int j = i; j >= 1; j--)
        {
            if (j == i || j == 1)
            {
                cout << "* ";
            }
            else
            {
                cout << "  ";
            }
        }

        cout << endl;
    }

    return 0;
}

File: Plot-the-Pattern/B_Chetanya_loves_triangles/Chetanya_loves_triangles.py
rows = int(input())

for i in range(1, rows + 1):
    for j in range(2 * rows - i, i, -1):
        print(" ", end="")

    for j in range(i, 0, -1):
        if j == i or j == 1:
            print("* ", end="")
        else:
            print("  ", end="")

    print() 

for i in range(rows - 1, 0, -1):
    for j in range(2 * rows - i, i, -1):
        print(" ", end="")

    for j in range(i, 0, -1):
        if j == i or j == 1:
            print("* ", end="")
        else:
            print("  ", end="")

    print()
